collector/collector_test.go

```diff
@@ -300,6 +309,158 @@ info:
 	})
 
 	Describe("Scan", func() {
+		Context("When users are created for the same stage on different files (issue kairos-io/kairos#1341)", func() {
```

Comment:
Add test to make sure https://github.com/kairos-io/kairos-sdk/pull/38 is addressed